src/mongo/MongoScrapbook.ts

```diff
@@ -294,12 +285,62 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 				parsedObject = [];
 			}
 		} else if (child instanceof mongoParser.FunctionCallContext || child instanceof ErrorNode) {
-			return {};
+			if (child.text.startsWith("ObjectId")) {
+				parsedObject = this.objectIdContextToObject(ctx, child.text);
+			} else {
+				return {};
```

Comment:
Should be doing an error instead of return {}.  Does your ISODate work fix that?